ansible/tests/enterprise/consul_namespaces.yml

```diff
@@ -0,0 +1,161 @@
+# User-story -> namespaces in Consul
+# Aim: show resource isolation of services
+# Reference: https://youtu.be/Ff6kLvKkJBE
+
+# 1. [Master token]
+#   1 Register service in namespace=default
+#   2 Create namespaces ["team1", "team2"]
+#   3 Generate admin tokens for namespaces ["team1", "team2"]
+# 2. [Admin team1 token]
+#   1. Verify write access, try register
+#   2. Verify read access, try list services
+
+- name: Verify that consul acl enabled and default_policy is deny
+  shell: consul acl policy list
+  register: consul_acl_default_policy_check
+  environment:
+    CONSUL_HTTP_TOKEN: ""
+  # must exit:1 -> 403 unauthorized
+  failed_when: consul_acl_default_policy_check.rc == 0
+
+# 1.1 Register service in namespace=default with m
+- name: Register service 'web' in namespace=default
+  shell: consul services register -name=web -address=1.1.1.1 -port=80
+  register: register_service_try_1
+  environment:
+    CONSUL_HTTP_TOKEN: master
+  failed_when: register_service_try_1.rc > 0
+
+- name: De-register service 'web' in namespace=default
+  shell: consul services deregister -namespace default -id=web
+  register: deregister_service_try_1
+  environment:
+    CONSUL_HTTP_TOKEN: master
+  failed_when: deregister_service_try_1.rc > 0
+
```

Comment:
Shouldn't the ansible tasks in this file be tagged as **test**?

Reply:
yeah. I think its good idea

Reply:
I do not see the the ansible tasks being tagged as **test** so wonder if you decided otherwise.